extension/src/popup/components/sendPayment/index.tsx

```diff
@@ -0,0 +1,274 @@
+import React, { useState, useEffect } from "react";
+import { useDispatch, useSelector } from "react-redux";
+
+import StellarSdk, { Asset } from "stellar-sdk";
+import { Types } from "@stellar/wallet-sdk";
+
+import { AccountBalancesInterface } from "@shared/api/types";
+import { signTransactionXDR } from "popup/ducks/access";
+import { AppDispatch } from "popup/App";
+import { settingsNetworkDetailsSelector } from "popup/ducks/settings";
+
+import { navigateTo } from "popup/helpers/navigate";
+import { ROUTES } from "popup/constants/routes";
+
+import { PopupWrapper } from "popup/basics/PopupWrapper";
+
+import { BackButton } from "popup/basics/Buttons";
+
+import "./styles.scss";
+
+interface CodeBalanceInterface {
```

Comment:
anyway we can make this interface name a bit more descriptive?

Reply:
was able to remove after this change 
https://github.com/stellar/freighter/pull/372/commits/a34a6c5e7a893fdbd95ffe13c7fb8a01deb8fc01